pkg/steps/git_source.go

```diff
@@ -0,0 +1,65 @@
+package steps
+
+import (
+	"context"
+	"fmt"
+
+	buildapi "github.com/openshift/api/build/v1"
+	imageclientset "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
+
+	"github.com/openshift/ci-operator/pkg/api"
+)
+
+type gitSourceStep struct {
+	config      api.ProjectDirectoryImageBuildStepConfiguration
+	resources   api.ResourceConfiguration
+	imageClient imageclientset.ImageV1Interface
+	buildClient BuildClient
+	jobSpec     *api.JobSpec
+}
+
+func (s *gitSourceStep) Inputs(ctx context.Context, dry bool) (api.InputDefinition, error) {
+	return s.jobSpec.Inputs(), nil
+}
+
+func (s *gitSourceStep) Run(ctx context.Context, dry bool) error {
+	return handleBuild(s.buildClient, buildFromSource(s.jobSpec, "", s.config.To, buildapi.BuildSource{
```

Comment:
That empty string looks a tiny bit suspicious -- what is it?

Reply:
its a From tag. In this case, I am not using it.